### PR TITLE
Added CFGD tag

### PR DESCRIPTION
**Pull request by shampur**
Signed-off-by: Charan Shampur <cshampur@cisco.com>

### CODE REVIEW

**Comment by dseevr** [comment]
@shampur could you `git reset HEAD~1` this and then `git add -p` (or use whatever tool you use) and do one commit per PR?  Having 4 PRs being addressed in 1 commit makes this extremely difficult to review... especially for people not familiar with the codebase (like me)

**Comment by shampur** [comment]
@dseevr Added separate commits. 291 and 353 are having similar changes so they are grouped under the same commit.

**Comment by dseevr** [comment]
@shampur awesome, thanks

File: app/components/models/contivglobals.ts [hunk 1]
```diff
@@ -44,6 +44,7 @@ export const ContivGlobals = {
     'NUMBER_REGEX' : '^[0-9]*$',
     'USERNAME_REGEX': /^([a-zA-Z0-9\_\-\.\@])+$/,
     'LDAPGROUP_REGEX': /([\=])+/g,
+    'NETWORK_NAME_REGEX': /^(([a-zA-Z0-9]|[a-zA-Z0-9][a-zA-Z0-9\-]*[a-zA-Z0-9])\.)*([A-Za-z0-9]|[A-Za-z0-9][A-Za-z0-9\-]*[A-Za-z0-9])$/,
```
Comment (dseevr) [inline app/components/models/contivglobals.ts]: Where did you get the requirements for this regex?

Also, you can remove the A-Z entries and just put a `i` on the end to make the regex case insensitive, e.g.,:

```js
/^(([a-z0-9]|[a-z0-9][a-z0-9\-]*[a-z0-9])\.)*([a-z0-9]|[a-z0-9][a-z0-9\-]*[a-z0-9])$/i
```

Saves a lot of typing lol

Comment (shampur) [inline app/components/models/contivglobals.ts]: https://github.com/contiv/contivmodel/blob/master/network.json#L66
This is the same regex net-plugin uses I guess.

Comment (dseevr) [inline app/components/models/contivglobals.ts]: I should've guessed lol

**Pull request merged**